src/poliastro/util.py

```diff
@@ -30,14 +30,14 @@ def time_range(start, *, periods=50, spacing=None, end=None, format=None, scale=
     ----------
     periods : int, optional
         Number of periods, default to 50.
-    spacing : Time or Quantity, optional
+    spacing : ~astropy.time.Time or astropy.units.Quantity, optional
```

Comment:
Please have consistency here as well 
```suggestion
    spacing : ~astropy.time.Time or ~astropy.units.Quantity, optional
```